app/views/users/show.html.erb

```diff
@@ -3,5 +3,18 @@
 
 <p>E-Mail: <%= @user.email %></p>
 
-
+<%= link_to 'New Study Session', '/study_sessions/new', method: :get %>
 <p><%= link_to 'Edit Profile', edit_user_path(@user) %></p>
+<p><%= link_to 'Delete Profile', "/users/#{@user.id}", method: :delete %></p>
+<% if current_user %>
+  <%= button_to "Connect to Spotify", spotify_login_path, class: "welcome-btn", method: :get %>
+<% end %>
+
+<% if @user.study_sessions.present? %>
+  <p>Previous Sessions</p>
+  <% @user.study_sessions.each do |session| %>
```

Comment:
According to travis this right here is failing lots of tests because it doesn't know what study_sessions are

Reply:
Got it fixed, these changes were in a seperate PR. I merged that branch into this one, for a cleaner master PR